site/src/app/Root.tsx

```diff
@@ -26,6 +28,12 @@ function Root({ apps }: Props) {
     const [showingAppDetails, setShowingAppDetails] = useState<AppDetails | null>(null);
     const [showingAboutDialog, setShowingAboutDialog] = useState(false);
 
+    let telemetry: Telemetry;
+
+    useEffect(() => {
+        telemetry = new Telemetry('InstrumentationKey=ae2167fd-0823-49e0-91a4-b6ec5a598490;IngestionEndpoint=https://northeurope-2.in.applicationinsights.azure.com/;LiveEndpoint=https://northeurope.livediagnostics.monitor.azure.com/;ApplicationId=bbb09627-f57e-40d4-a59a-50c1e0243bae');
+    }, []);
```

Comment:
Consider a singleton so such as getTelemetry().

As it is it should only be created one but you can create the obj only when needed i.e line 76